client.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2020 Chair of Applied Cryptography, Technische Universität
+// Darmstadt, Germany. All rights reserved. This file is part of
+// perun-eth-demo. Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package prnm
+
+import (
+	srand "crypto/rand"
+	"fmt"
+	"log"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/pkg/errors"
+
+	"perun.network/go-perun/apps/payment"
+	ethchannel "perun.network/go-perun/backend/ethereum/channel"
+	ethwallet "perun.network/go-perun/backend/ethereum/wallet"
+	"perun.network/go-perun/channel"
+	"perun.network/go-perun/client"
+	"perun.network/go-perun/peer/net"
+	"perun.network/go-perun/wallet"
+)
+
+type Client struct {
+	cfg *Config
+
+	client *client.Client
+
+	w       *ethwallet.Wallet
+	onChain wallet.Account
+
+	dialer *net.Dialer
+}
+
+func NewClient(cfg *Config) (*Client, error) {
+	//acc := unlockAccount(cfg)
+	//var w wallet.Wallet
```

Comment:
We probably should open an issue because right now i import a secret key instead of unlocking the wallet. This still works because it can be pre-funded by `ganache` and does not change the API.  
I could not test it with a real wallet yet because of how Android storage works.

Reply:
We discussed to create a new function
```go
prnm.CreateAccount(walletPath, passphrase string) (account string, err error)
```
that opens or creates a wallet path and then generates a new account in it and returns its address.